integrations/standalone/tests/mock/browser.spec.ts

```diff
@@ -34,6 +34,32 @@ test.describe('Script browser', () => {
     await expect(code(page).getByRole('textbox')).toHaveValue('test 123 <%=in.bla%>');
   });
 
+  test('browser init searchfilter', async ({ page }) => {
+    const inscriptionView = await InscriptionView.mock(page);
+    const task = inscriptionView.accordion('Call');
+    await task.toggle();
+
+    await page.getByText('Code').click();
+    const codeField = task.scriptArea();
+
+    await codeField.focus();
+    await assertCodeVisible(page);
+    await page.keyboard.down('Control');
+    await page.keyboard.press('A');
+    await page.keyboard.up('Control');
+
+    await page.keyboard.press('Delete');
+
+    await page.keyboard.type('handle');
```

Comment:
Maybe 'control' will not work in webkit, because webkit simulates mac a bit and on mac it is 'cmd+A', btw 'Meta'.

But I think the codeField (ScriptArea pageobject) has a clear of a fill method to clean/fill content... Why you don't reuse them?